### DISCUSSION TITLE
ExtendedPanID must be 8 digits long, got 1

### DISCUSSION THREAD

**Discussion by TallDragon** [Koenkk/zigbee2mqtt#23193] [Devices]
Guys
I have the CC2652RB in use for Zigbee, after the latest power outage Zigbee2MQTT does not start anymore in my docker. 
I get:
![image](https://github.com/Koenkk/zigbee2mqtt/assets/36789802/4b4d1cd0-0537-437c-9468-62bda3f85add)

Now I replaced with the Sonos 3.0, removed  the database and json file. Still getting the same result.

It was working flawless for the last 4 months. Help??

**Comment by Koenkk** [comment]
Looks at your `data/configuration.yaml`, it seems you `ext_pan_id` is invalid
